src/npc.h

```diff
@@ -554,7 +554,7 @@ class npc : public player
  ~npc() override;
 
     /** Construct an NPC with standard stats */
-    static npc standard();
+    static npc make_standard();
```

Comment:
I don't want to confound it with creation of fake NPC's which I agree are a bad idea. The purpose of the named constructor is I want to define a standard NPC for test cases which can be updated in one place if our definition of standard varies.